src/useSpeechRecognition.js

```diff
@@ -24,6 +25,14 @@ const useSpeechRecognition = (props = {}) => {
     onResult(transcript);
   };
 
+  const handleError = (event) => {
+    if (event.error === "not-allowed") {
```

Comment:
Minor: there are a couple of linting errors with this block. If you run `yarn lint` you'll see what I'm talking about

Reply:
Hopefully I got them all. On my (Windows) machine, yarn lint complains incessantly about my line endings being CRLF rather than LF, which makes it hard to pick out the true issues. I tried changing default line endings in VS Code but it didn't really help without my going through each file to update existing line endings.